lore/settings.py

```diff
@@ -104,7 +106,7 @@ def get_var(name, default):
     'guardian.backends.ObjectPermissionBackend',
 )
 
-ROOT_URLCONF = 'ui.urls'
+ROOT_URLCONF = 'lore.urls'
```

Comment:
The rest app has its own urls so it made sense to move the base for urls back into the lore app and then include ui and rest urls from there.